tests/async/test_evaluate.py

```diff
@@ -216,17 +216,30 @@ async def test_evaluate_evaluate_date(page: Page) -> None:
     result = await page.evaluate(
         '() => ({ date: new Date("2020-05-27T01:31:38.506Z") })'
     )
-    assert result == {"date": datetime.fromisoformat("2020-05-27T01:31:38.506")}
+    assert result == {
+        "date": datetime.fromisoformat("2020-05-27T01:31:38.506").replace(
+            tzinfo=timezone.utc
+        )
+    }
 
 
 async def test_evaluate_roundtrip_date(page: Page) -> None:
     date = datetime.fromisoformat("2020-05-27T01:31:38.506")
     result = await page.evaluate("date => date", date)
+    assert result.timestamp() == date.timestamp()
```

Comment:
Why do we compare only timestamps, is `result != date`?

Reply:
Because `datetime.fromisoformat("2020-05-27T01:31:38.506")` has `tzinfo=None` while result has `tzinfo=UTC` so the strict assert would fail.